static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
+import config from 'lib/config';
+
+let dm;
+let DotMetricsObj;
+const IpsosTagging = function () {
+
+    console.log("Ipsos tag fired");
```

Comment:
do these logs need to be present full-time?

Reply:
These are just for testing purposes at the moment

Reply:
It might be preferable to use `console.debug` in that case!